docker-java-api/src/main/java/com/github/dockerjava/api/model/Image.java

```diff
@@ -26,22 +29,38 @@
     private String parentId;
 
     @JsonProperty("RepoTags")
-    private String[] repoTags;
+    private List<String> repoTags;
+
+    @JsonProperty("RepoDigests")
+    private List<String> repoDigests;
 
     @JsonProperty("Size")
     private Long size;
 
     @JsonProperty("VirtualSize")
     private Long virtualSize;
 
+    @JsonProperty("SharedSize")
+    private Long sharedSize;
+
+    @JsonProperty("Labels")
+    public Map<String, String> labels;
+
+    @JsonProperty("Containers")
+    private Integer containers;
+
     public String getId() {
         return id;
     }
 
-    public String[] getRepoTags() {
+    public List<String> getRepoTags() {
```

Comment:
please avoid breaking changes on public APIs

Reply:
Contribution doc  mention prefer to use Collections i had idea convert fields, but keep array getter with deprecated mark

Reply:
@saffis yes, new fields can be `List<String>`, just we can't change the return type of existing getters